epub34/a11y/index.html

```diff
@@ -1613,13 +1600,12 @@ <h2>Optimized publications</h2>
 				<li>The EPUB publication should identify the standard or guidelines it follows in a
 						<code>conformsTo</code> property in accordance with [[dcterms]] so this information can be made
 					available to users.</li>
-				<li>If the standard does not define conformance values for the <code>conformsTo</code> property, EPUB
-					creators should use a URL [[url]] to where the standard is publicly available so users can look up
-					the specific details of the standard.</li>
+				<li>If the standard does not define conformance values for use in the <code>conformsTo</code> property,
+					the <code>conformsTo</code> property should include a URL [[url]] to where the standard is publicly
```

Comment:
SHOULD caps?

Reply:
No, the whole section on optimized publications is informative so we can't have normative keywords here.

I wonder if this should really be an appendix, or even split off into its own note? Unlike the informative distribution, which still affects accessible publications, this classification of optimized publications is kind of an aside.